### PR TITLE
Updating maintainers list, adding Peter

### PR DESCRIPTION
**Pull request by sumobrian**
### Description
Adding @peternied to list of maintainers and @kartg to emeritus list.

### Check List
- [ ] New functionality includes testing
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: MAINTAINERS.md
Comment (peternied) [inline MAINTAINERS.md:16]:
```suggestion
| Peter Nied         | [peternied](https://github.com/peternied)               | Amazon      |
```

**Review by AndreKurait — APPROVED** [review]
Approved, please add peter to write access to repo and remove karthg before merging this PR

**Comment by AndreKurait** [comment]
> Approved, please add peter to write access to repo and remove karthg before merging this PR

@opensearch-project/admin

**Comment by bbarani** [comment]
Removed @kartg  and added @peternied as one of the maintainer of this repo.

**Comment by peternied** [comment]
- Bypassing the failing build check by creating https://github.com/opensearch-project/opensearch-migrations/issues/630

**Pull request merged**
